disnake/ext/commands/converter.py

```diff
@@ -431,9 +431,9 @@ async def convert(self, ctx: AnyContext, argument: str) -> disnake.Message:
         try:
             return await channel.fetch_message(message_id)
         except disnake.NotFound:
-            raise MessageNotFound(argument)
+            raise MessageNotFound(argument) from None
         except disnake.Forbidden:
-            raise ChannelNotReadable(channel)  # type: ignore
+            raise ChannelNotReadable(channel) from None  # type: ignore
```

Comment:
Maybe consider the original error here as well? Not sure.

Reply:
I don't think its worth keeping as these are just reskinned 403s and 404s